.circleci/markdown

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+set -euv
+
+FILES=$(find . -name '*.md' | grep -v node_modules)
+for file in $FILES
+do
+  echo "============== checking $file"
+  npx markdownlint-cli "$file"
+done
```

Comment:
Can you put a new line at the end of this file please ❤️

Reply:
done